subprojects/docs/src/samples/announce/groovy/build.gradle

```diff
@@ -13,14 +13,16 @@ announce {
 
 
 // tag::announce-usage[]
-task helloWorld {
+def helloWorld = tasks.register("helloWorld") {
     doLast {
         println "Hello, world!"
     }
 }
 
-helloWorld.doLast {
-    announce.announce("helloWorld completed!", "twitter")
-    announce.announce("helloWorld completed!", "local")
+helloWorld.configure {
```

Comment:
Should we capture the `TaskProvider` in a variable or use `tasks.named` call each time? Of course, some sample that extracts part of the script inside the user guide should just use `task.named` to have full context.